tests/convention/Job/LoggerTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yokai\Batch\Sources\Tests\Convention\Job;
+
+use Exception;
+use PHPUnit\Framework\TestCase;
+use Yokai\Batch\Factory\UniqidJobExecutionIdGenerator;
+use Yokai\Batch\JobExecution;
+
+class LoggerTest extends TestCase
```

Comment:
The test class is not in the right dir, this is why Codecov is not seeing the coverage

`tests/convention/Job/LoggerTest.php` instead of `src/batch/tests/Job/LoggerTest.php`